extension/src/stats_agg.rs

```diff
@@ -1521,25 +1749,34 @@ mod tests {
                 state.failed_msg(PRINT_VALS)
             );
         }
+
+        if do_moving_agg {
+            approx::assert_relative_eq!(
+                pg_moving_agg_result.unwrap(),
+                tk_moving_agg_result.unwrap(),
+                max_relative = 1e-9,
+            )
+        }
     }
 
     fn pg1d_aggx(agg: &str) -> String {
-        format!("SELECT {}(test_x) FROM test_table", agg)
+        format!("SELECT {agg}(test_x), (SELECT {agg}(test_x) OVER (ORDER BY test_x ROWS BETWEEN 1 PRECEDING AND 1 FOLLOWING) FROM test_table LIMIT 1 OFFSET 3) FROM test_table", agg = agg)
     }
 
     fn pg1d_aggy(agg: &str) -> String {
-        format!("SELECT {}(test_y) FROM test_table", agg)
+        format!("SELECT {agg}(test_y), (SELECT {agg}(test_y) OVER (ORDER BY test_x ROWS BETWEEN 1 PRECEDING AND 1 FOLLOWING) FROM test_table LIMIT 1 OFFSET 3) FROM test_table", agg = agg)
     }
 
     fn pg2d_agg(agg: &str) -> String {
-        format!("SELECT {}(test_y, test_x) FROM test_table", agg)
+        format!("SELECT {agg}(test_y, test_x), (SELECT {agg}(test_y, test_x) OVER (ORDER BY test_x ROWS BETWEEN 1 PRECEDING AND 1 FOLLOWING) FROM test_table LIMIT 1 OFFSET 3) FROM test_table", agg = agg)
```

Comment:
I don't think this is going to be enough rows to actually see a difference and possibly will never even call the twofloat version because the condition that stops us from using the moving agg will be triggered here, I think we probably need to test with a bit more data...

Reply:
I made the floating point error threshold `f64::INFINITY` in test mode – hopefully that should do the trick here?